LunrCore/Polyfills/StringExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿#if NETSTANDARD2_0
+namespace Lunr
+{
+    internal static class StringExtensions
+    {
+        public static bool Contains(this string text, char value)
+        {
+            return text.IndexOf(value) > -1;
+        }
+    }
+}
+#endif
```

Comment:
The benefits of this new method seem extremely slim. Can't we just leave the IndexOfs in place? Are there any benefits besides the slight readability improvement?